devs/docs/es-backports.rst

```diff
@@ -156,31 +157,31 @@ should be crossed out as well.
 - [ ] 3d69b5c41e2 Introduce point in time APIs in x-pack basic (#61062)
 - [ ] 7b941a18e9d Optimize Snapshot Shard Status Update Handling (#62070) (#62219)
 - [ ] 6710104673d Fix Creating NOOP Tasks on SNAPSHOT Pool (#62152) (#62157)
-- [ ] ed4984a32e7 Remove Redundant Stream Wrapping from Compression (#62017) (#62132)
+- [x] ed4984a32e7 Remove Redundant Stream Wrapping from Compression (#62017) (#62132)
 - [x] 075271758e3 Keep checkpoint file channel open across fsyncs (#61744)
 - [ ] 2bb5716b3dc Add repositories metering API (#62088)
 - [ ] bb0a583990e Allow enabling soft-deletes on restore from snapshot (#62018)
 - [ ] 3389d5ccb25 Introduce integ tests for high disk watermark (#60460)
 - [ ] 395538f5083 Improve Snapshot State Machine Performance (#62000) (#62049)
-- [ ] a295b0aa86f Fix null_value parsing for data_nanos field mapper (#61994)
-- [ ] 1799c0c5833 Convert completion, binary, boolean tests to MapperTestCase (#62004)
-- [ ] 0c8b4385777 Add support for runtime fields (#61776)
+- [s] a295b0aa86f Fix null_value parsing for data_nanos field mapper (#61994)
+- [s] 1799c0c5833 Convert completion, binary, boolean tests to MapperTestCase (#62004)
+- [s] 0c8b4385777 Add support for runtime fields (#61776)
 - [ ] b26584dff89 Remove unused deciders in BalancedShardsAllocator (#62026)
-- [ ] 6d08b55d4e3 Simplify searchable snapshot shard allocation (#61911)
-- [ ] 66bb1eea982 Improve error messages on bad [format] and [null_value] params for date mapper (#61932)
-- [ ] 31c026f25cc upgrade to Lucene-8.7.0-snapshot-61ea26a (#61957) (#61974)
-- [ ] af01ccee93e Add specific test for serializing all mapping parameter values (#61844) (#61877)
-- [ ] d59343b4ba8 Allow [null] values in [null_value] (#61798) (#61807)
+- [s] 6d08b55d4e3 Simplify searchable snapshot shard allocation (#61911)
+- [s] 66bb1eea982 Improve error messages on bad [format] and [null_value] params for date mapper (#61932)
+- [s] 31c026f25cc upgrade to Lucene-8.7.0-snapshot-61ea26a (#61957) (#61974)
+- [s] af01ccee93e Add specific test for serializing all mapping parameter values (#61844) (#61877)
+- [s] d59343b4ba8 Allow [null] values in [null_value] (#61798) (#61807)
 - [ ] 3fd25bfa877 Fix Concurrent Snapshot Create+Delete + Delete Index (#61770) (#61773)
 - [s] 5723b928d7d Remove Outdated Snapshot Docs (#61684) (#61728)
-- [ ] 1bfebd54ea7 [7.x] Allocate newly created indices on data_hot tier nodes (#61342) (#61650)
+- [s] 1bfebd54ea7 [7.x] Allocate newly created indices on data_hot tier nodes (#61342) (#61650)
 - [s] f769821bc80 Pass SearchLookup supplier through to fielddataBuilder (#61430) (#61638)
 - [ ] b866aaf81c0 Use int for number of parts in blob store (#61618)
-- [ ] 9f566644af2 Do not create two loggers for DeprecationLogger backport(#58435)  (#61530)
```

Comment:
Skipped `DeprecationLogger` related patches, we did a lot of change there in https://github.com/crate/crate/commit/7ac48662cc8015b6ee14f063ca67daccc72c1bda